orchagent/port.h

```diff
@@ -100,7 +100,7 @@ class Port
     uint32_t            m_mtu = DEFAULT_MTU;
     uint32_t            m_speed = 0;    // Mbps
     std::string         m_learn_mode = "hardware";
-    bool                m_autoneg = false;
+    int                 m_autoneg = -1;  // -1 means not set
```

Comment:
Suggest adding meanings of other values (e.g., `// -1 means not set, 0 = disabled, 1 = enabled`) in the comment for posterity.